CTRE/LowLevel/Device_LowLevel.cs

```diff
@@ -348,9 +345,22 @@ private ErrorCode ConfigSetParameterRaw(ParamEnum paramEnum, int value, byte sub
             /* send it */
             uint arbId = PARAM_SET | GetDeviceNumber();
             int status = CTRE.Native.CAN.Send(arbId, frame, 8, 0);
+            /* Send Error Exists when Timeout is 0ms and doing continuous configs, or high CAN Utiliztion */
+            if (status != 0)
+            {
+                /* Retry until success or we have exceeded 1000 attempts. */
+                for (int i = 0; i < 1000; i++)
```

Comment:
1000 is crazy, how about a max of 5?

Reply:
At 5 attempts, we fail after about every 300 Configs, returning a -1 ErrorCode. I'm not sure if we want to recreate a more realistic scenario and be fine with that or test worse case and have some massive number (my 1000) to always succeed at the CAN.Send

Reply:
I added a sleep(1) in my version - that should be enough time (5-ms) for a frame slot to become free.  I'm not sure how much time 1000 fast calls are.